draft-ietf-quic-recovery.md

```diff
@@ -213,14 +213,17 @@ or negotiated in order to better suit a variety of environments.
   a packet lost.  In fraction of an RTT.
 
 * kMinTLPTimeout: 10ms
- Minimum time in the future a tail loss probe alarm may be set for.
+  Minimum time in the future a tail loss probe alarm may be set for.
```

Comment:
Editorial: this might look OK in markdown, but it turns into a real mess in the text version.  I think that you want something like the following:

```
kMinTLPTimeout (default 10ms):
 : Minimum time in the future [...]
```